src/seq/UndoBridge.cpp

```diff
@@ -75,9 +75,11 @@ using SeqAction1 = SeqAction<MidiSequencerPtr, SqCommand, SequencerModule, Seque
 using SeqAction4 = SeqAction<MidiSequencer4Ptr, Sq4Command, Sequencer4Module, Sequencer4Widget>;
 #endif
 
-void UndoRedoStack::setModuleId(int id)
-{
-    this->moduleId = id;
+void UndoRedoStack::setModuleId(int64_t id, bool fromUI)
+{   
+    if (id != this->moduleId) {
+         this->moduleId = id;
+    }
```

Comment:
added a param just to help with debugging. Only save if different (again, for debugging).